sdk/core/core-rest-pipeline/package.json

```diff
@@ -93,7 +93,7 @@
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-auth": "^1.3.0",
-    "@azure/core-tracing": "1.0.0-preview.12",
+    "@azure/core-tracing": "1.0.0-preview.11",
```

Comment:
Wanted to call out this change, which I discovered in #15881 - because core-client is still depending on core-rest-pipeline 1.0.3 we cannot take this change in yet. Downgrading here means that all corev2 packages are on the same version of core-tracing.

I am hopeful though that by this release we can take core-rest-pipeline 1.1.0 GA and put all our dependencies on the same version!